test/explorer/data_frame/parquet_test.ex

```diff
@@ -0,0 +1,82 @@
+defmodule Explorer.DataFrame.ParquetTest do
+  @moduledoc """
+  Integration tests, based on `Explorer.DataFrame.CsvTest`.
+  """
+  use ExUnit.Case, async: true
+  alias Explorer.DataFrame, as: DF
+  alias Explorer.Series
+
+  # https://doc.rust-lang.org/std/primitive.f64.html#associatedconstant.EPSILON
+  @f64_epsilon 2.2204460492503131e-16
+
+  def tmp_file!(data) do
+    filename = System.tmp_dir!() |> Path.join("data.parquet")
+    :ok = DF.to_parquet(data, filename)
+    filename
+  end
```

Comment:
Just another way to write. But the main point is rename `data` by `df`. WDYT?

```suggestion
  def tmp_file!(df) do
    System.tmp_dir!() |> Path.join("data.parquet") |> tap(fn filename ->
      :ok = DF.to_parquet(df, filename)
    end)
  end
```

Reply:
100% agree!
Also, always in favour of saving some variables 👍